labelbox/client.py

```diff
@@ -648,7 +649,9 @@ def get_data_row_metadata_ontology(self) -> DataRowMetadataOntology:
             DataRowMetadataOntology: The ontology for Data Row Metadata for an organization
 
         """
-        return DataRowMetadataOntology(self)
+        if self.data_row_metadata_ontology is None:
```

Comment:
Good point! Currently, the way customers would be using this object is, they'll call
```mdo = client.get_data_row_metadata_ontology()
mdo.fields```
and if they happen to change the metadata after, mdo.fields wouldn't return the most up-to-date metadata, because the `fields` is only set when `client.get_data_row_metadata_ontology()` is first called, via `DataRowMetadataOntology`'s `init`.

They would be able to update the `fields` only if they call `client.get_data_row_metadata_ontology()` again.

In order to support fetching of the latest metadata, I can make a change to make `refresh_ontology` public, so the customers have a way to refresh the metadata list. What do you think?

Reply:
Good point! Currently, the way customers would be using this object is, they'll call
```
mdo = client.get_data_row_metadata_ontology()
mdo.fields
```
and if they happen to change the metadata after, mdo.fields wouldn't return the most up-to-date metadata, because the `fields` is only set when `client.get_data_row_metadata_ontology()` is first called, via `DataRowMetadataOntology`'s `init`.

They would be able to update the `fields` only if they call `client.get_data_row_metadata_ontology()` again.

In order to support fetching of the latest metadata, I can make a change to make `refresh_ontology` public, so the customers have a way to refresh the metadata list. What do you think?

Reply:
I like having `refresh_ontology` as public!

Reply:
Done!